jira/cli/templates.go

```diff
@@ -52,12 +52,18 @@ comments:
 `
 const default_edit_template = `# issue: {{ .key }}
 update:
+{{if .overrides.comment}}
   comment:
-    - add: 
+    - add:
         body: |~
           {{ or .overrides.comment "" | indent 10 }}
+{{end}}
+{{ if .overrides.label }}
+  labels:
+    - add: {{ .overrides.label }}
+{{end}}
 fields:
-  summary: {{ or .overrides.summary .fields.summary }}
+  summary: "{{ or .overrides.summary .fields.summary }}"
```

Comment:
Okay make sense.  I guess I prefer this syntax:
```
summary: >-
    {{ or .overrides.summary .fields.summary }}
```

I have tried to move all extraneous formatting characters (ie quotes, etc)